kaspad/src/args.rs

```diff
@@ -306,7 +311,7 @@ pub fn cli() -> Command {
             .require_equals(true)
             .value_parser(clap::value_parser!(u64))
             .help("Interval in seconds for performance metrics collection."),
-    );
+    ).arg(arg!(--upnp "Enable upnp"));
```

Comment:
Can we make upnp enabled by default?

Then flag here can be `--disable-upnp` which disables the implementation. Ideally, the base `kaspad --utxoindex` will do the upnp check.